src/pretix/plugins/sendmail/views.py

```diff
@@ -183,6 +199,9 @@ def get_context_data(self, **kwargs):
             log.pdata['items'] = [
                 itemcache[i['id']] for i in log.pdata.get('items', [])
             ]
+            log.pdata['checkin_lists'] = [
+                checkin_list_cache[i['id']] for i in log.pdata.get('checkin_lists', [])
```

Comment:
You could maybe add a ``if i['id'] in checkin_list_cache`` to prevent crashes if a check-in list got deleted in the meantime

Reply:
Will do